src/components/screens/SUE/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './SueDetailScreen';
+export * from './SueListScreen';
+export * from './SueMapScreen';
```

Comment:
the files are still named SUE...instead of Sue...?

Reply:
I don't think so. Or am I seeing it wrong. I'm seeing it as `Sue...`

Reply:
I see. These changes are not being recognised by Fork! That's the reason for this.

Reply:
by git you mean. you remember filesystem renaming complications with git?

Reply:
Yeah, I'm talking about Git. I'm sorry, but I don't remember that.
But it should be fixed now.
https://github.com/smart-village-solutions/smart-village-app-app/pull/960/commits/9d49e470ef529e81892edcb2620c8c7002aa8855

Reply:
yes looks good in here

<img width="199" alt="image" src="https://github.com/smart-village-solutions/smart-village-app-app/assets/1942953/2b40440a-f52c-491c-a877-e7585cc819a1">